### PR TITLE
chore: update README.md

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,26 @@
-# <img src="projects/demo/src/assets/images/taiga.svg" alt="taiga ui logo" width="48px"> Taiga UI
+<center>
 
+# <img src="projects/demo/src/assets/images/taiga.svg" alt="taiga ui logo" width="36px"> Taiga UI
+
+[![CodeFactor](https://www.codefactor.io/repository/github/taiga-family/taiga-ui/badge)](https://www.codefactor.io/repository/github/ng-zorro/ng-zorro-antd)
```
Comment (waterplea) [inline README.md:5]: ng-zorro everywhere :D

File: README.md [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
 [![npm version](https://img.shields.io/npm/v/@taiga-ui/cdk.svg)](https://npmjs.com/package/@taiga-ui/cdk)
 [![codecov](https://codecov.io/gh/tinkoff/taiga-ui/branch/main/graphs/badge.svg)](https://app.codecov.io/gh/Tinkoff/taiga-ui/tree/main/projects)
 [![All packages CI](https://github.com/taiga-family/taiga-ui/actions/workflows/build.yml/badge.svg?branch=main)](https://github.com/taiga-family/taiga-ui/actions/workflows/build.yml)
+[![GitHub Release Date](https://img.shields.io/github/release-date/taiga-family/taiga-ui.svg)](https://github.com/NG-ZORRO/ng-zorro-antd/releases)
```
Comment (waterplea) [inline README.md:9]: Let's move it next to npm

File: README.md (hunk 1 above)
Comment (splincode) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (splincode) [reply README.md]: fixed

File: README.md [hunk 3]
```diff
@@ -8,11 +11,15 @@
 [![Discord](https://img.shields.io/discord/748677963142135818?color=7289DA&label=%23taiga-ui&logo=discord&logoColor=white)](https://discord.gg/Us8d8JVaTg)
 [![angular-open-source-starter](https://img.shields.io/badge/made%20with-angular--open--source--starter-d81676?logo=angular)](https://github.com/taiga-family/angular-open-source-starter)
```
Comment (waterplea) [inline README.md:12]: We should probably remove it

**Review by waterplea — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**